opentracing/span.py

```diff
@@ -63,18 +62,17 @@ def __exit__(self, exc_type, exc_val, exc_tb):
                                             'tb': exc_tb})
         self.finish()
 
-    def start_child(self, operation_name):
+    def start_child(self, operation_name, tags=None):
```

Comment:
Booked https://github.com/opentracing/api-python/issues/4 to continue this particular discussion.